Source/FuseJS/Tests/Base64.Test.uno

```diff
@@ -11,16 +12,16 @@ namespace FuseJS.Test
 		[Test]
 		public void JavaScriptTests()
 		{
-			new FuseJS.Base64();
-			using (var jsRecorder = new JsTestRecorder())
-			{
-				var context = jsRecorder.Begin();
-				var moduleResult = new FileModule(import("Base64Tests.js")).Evaluate(context, "Base64Tests");
-				if (moduleResult.Error != null)
-					throw moduleResult.Error;
+			var test = new JSTest(JavaScriptTestsInner);
+			test.WaitOnResults();
```

Comment:
Looks like
```
var test = new JSTest(foo);
test.WaitOnResults();
```
is a common pattern now. How about making a helper in JSTest, along the lines of 
```uno
static void RunTest(Action<Fuse.Scripting.Context> method)
{
	var test = new JSTest(method);
	test.WaitOnResults();
}
```
...to reduce the code replication a bit?

Reply:
True, I'll do that now

Reply:
done